emerald-lang-spec/2num-operators.em

```diff
@@ -0,0 +1,32 @@
+ (+ 5 4)
+ ; compiles to:: 5 + 4
```

Comment:
It might be difficult and fragile to make assertions about the compiled output, but we could run the program and make assertions about what it prints or returns :)